its/pom.xml

```diff
@@ -122,9 +98,11 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <trimStackTrace>false</trimStackTrace>
+          <failIfNoTests>true</failIfNoTests>
```

Comment:
@gregory-paidis-sonarsource I think you have a history with ITs that are secretly ignored if I remember correctly. Can you add this magic line to other projects?